src/main/java/mate/academy/security/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.security;
+
+import mate.academy.exception.AuthenticationException;
+import mate.academy.lib.Dao;
+import mate.academy.lib.Inject;
+import mate.academy.model.Driver;
+import mate.academy.service.DriverService;
+
+@Dao
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Inject
+    private DriverService driverService;
```

Comment:
I'm not sure if we need to inject a service into another service. I think there should be DriverDao. just FYI, I don't demand this change